Add more flags to WebView2 documentation (#3148)
MicrosoftDocs/edge-developer: microsoft-edge/webview2/concepts/webview-features-flags.md

| `msFloatyShouldHonorIndiaHoldout` | If `true`, honors the India holdout group.  Use this flag, set to `false`, to disable the Floaty feature that's enabled if the user is part of the India holdout group, because WebView doesn't support browser retention experiments. |
| `msOverlayScrollbarWinStyle` | Whether the users can change between overlay and non-overlay modes for Fluent scrollbars. |
| `msPdfEnableAsPreview` | This features enables the PDF viewer to launch with a minimal toolbar and in read-only preview mode. |

> | `msSingleSignOnOSForPrimaryAccountIsShared` | If enabled, allows implicit sign in to Microsoft web pages on any account using the primary os account information. |

Reviewer: ```suggestion
| `msSingleSignOnOSForPrimaryAccountIsShared` | If enabled, allows implicit sign-in to Microsoft webpages using any account, by using the information from the primary OS account. |
```
tech review this suggested change, re: accurate meaning.  @monica-ch 

https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/sign-in-sign-out - IMO, "implicit sign-in" works in this case.
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/o/operating-environment-operating-system - doesn't mention "OS", but I think it ought to, b/c that's more familiar in colloq. than "operating system".
